src/Model/ItemListPage.php

```diff
@@ -4,40 +4,61 @@
 
 namespace Libero\ContentApiBundle\Model;
 
+use ArrayIterator;
 use Countable;
-use IteratorAggregate;
-use Traversable;
-use function count;
+use Iterator;
+use function is_array;
+use function iterator_count;
 
-final class ItemListPage implements Countable, IteratorAggregate
+final class ItemListPage implements Countable, Iterator
```

Comment:
Alternative is for https://github.com/libero/content-api-bundle/blob/66ee2b7f57649c7ac123e78a05d8dde0cfbe5180/src/Test/ItemsTest.php#L226-L229 to wrap them in an `IteratorIterator`.

Reply:
If you want to avoid defining too many methods; but it seems this version keeps it simple rather than creating boilerplate adapter objects like `IteratorIterator`s